packages/sdk/src/internal/External/Idle.ts

```diff
@@ -39,7 +18,7 @@ export async function getGovTokensAmounts(
   const tokensAmounts = await Promise.all(
     amounts.map(async (amount, index) => {
       const token = await Viem.readContract(client, args, {
-        abi: getGovTokensAmountsAbi,
+        abi: parseAbi(["function govTokens(uint256) view returns (address)"]),
```

Comment:
Let's move that out o the `map`. This parses it in a loop.

Reply:
Good point, fixed.